tools/ign_TEST.cc

```diff
@@ -89,6 +90,30 @@ TEST(CmdLine, MsgInfo)
     << output;
 }
 
+/////////////////////////////////////////////////
+TEST(CmdLine, MsgHelpVsCompletionFlags)
+{
+  // Flags in help message
+  auto output = custom_exec_str("ign msg --help --force-version " + g_version);
+  EXPECT_NE(std::string::npos, output.find("--info")) << output;
+  EXPECT_NE(std::string::npos, output.find("--list")) << output;
+  EXPECT_NE(std::string::npos, output.find("--help")) << output;
+  EXPECT_NE(std::string::npos, output.find("--force-version")) << output;
+  EXPECT_NE(std::string::npos, output.find("--versions")) << output;
+
+  // Flags in bash completion
+  std::ifstream scriptFile(std::string(PROJECT_SOURCE_PATH) +
+    "/src/cmd/msgs.bash_completion.sh");
```

Comment:
`ignition::common::JoinPaths`

Reply:
This package also does not have dependency on ign-common. There's a comment like this in another source file:

https://github.com/gazebosim/gz-msgs/blob/7e1b8d463ad00430e046f997bcce64315838ba9b/src/Utility.cc#L37-L39

I don't know if we should do a `QUIET` find like in gz-plugin or hardcode the `/` like tests there.

Reply:
744a6a2 uses `std::filesystem::path`